Xamarin.Forms.Core/Registrar.cs

```diff
@@ -265,21 +266,21 @@ public static void RegisterRenderers(HandlerAttribute[] attributes)
 
 		public static void RegisterStylesheets()
 		{
-			var assembly = typeof(StyleSheets.StylePropertyAttribute).GetTypeInfo().Assembly;
+			var assembly = typeof(StylePropertyAttribute).GetTypeInfo().Assembly;
 
 #if NETSTANDARD2_0
-			object[] styleAttributes = assembly.GetCustomAttributes(typeof(StyleSheets.StylePropertyAttribute), true);
+			object[] styleAttributes = assembly.GetCustomAttributes(typeof(StylePropertyAttribute), true);
 #else
 			object[] styleAttributes = assembly.GetCustomAttributes(typeof(StyleSheets.StylePropertyAttribute)).ToArray();
```

Comment:
A small detail, after adding the namespace, `StyleSheets` is no longer necessary here.

`object[] styleAttributes = assembly.GetCustomAttributes(typeof(StylePropertyAttribute)).ToArray();`